typescript/scripts/src/copy-supabase-bucket.ts

```diff
@@ -0,0 +1,122 @@
+import { createClient, SupabaseClient } from "@supabase/supabase-js";
+import Bluebird from "bluebird";
```

Comment:
Maybe it could be nice to give a bit of context for why this script exists on the repository.
A few lines of comments and a link to this issue/PR could do the job. 

I think it's important to mention that this script isn't used on a regular basis, it was more a one-shot.

Reply:
@nacho-carnicero @RaynalHugo It's dead code then?

Reply:
I'd say it's not dead code, rather it's rather a script that was done to execute a punctual action that we might do again in the future (so I thought we would like to save it)

Reply:
Added comment in b47cd1d055e6e9b7dec0c24c321094438ddb1232

Reply:
👌